src/mad_spark_alt/core/qadi_prompts.py

```
@@ -0,0 +1,187 @@
+3. [Future scenario where this would be effective]
+
+Conclusion: [Is our answer universally applicable, context-specific, or needs refinement?]
+"""
+
+
+# Phase-specific hyperparameters
+PHASE_HYPERPARAMETERS = {
+    "questioning": {
+        "temperature": 0.3,   # Low - need focused, precise question
+        "max_tokens": 150,    # Short - just one question
+        "top_p": 0.9
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)